testing/rpm2cpio/APKBUILD

```diff
@@ -0,0 +1,22 @@
+# Contributor: TBK <alpine@jjtc.eu>
+# Maintainer: TBK <alpine@jjtc.eu>
+pkgname=rpm2cpio
+pkgver=1.4
+pkgrel=0
+pkgdesc="Convert .rpm files to cpio format"
+url="https://www.freshports.org/archivers/rpm2cpio/"
+arch="noarch"
+license="BSD-2-Clause"
+options="!check" # No test suite
+depends="libarchive-tools"
+source="rpm2cpio.sh"
+
+build() {
```

Comment:
This `build` function can be removed when it does nothing.

Reply:
@Ikke Fixed.